playground/json-stringify/vite.config.js

```diff
@@ -0,0 +1,3 @@
+import { defineConfig } from 'vite'
+
+export default defineConfig({ json: { stringify: true } })
```

Comment:
Having a copy of playground might be overkill. Can you add a specific test only focused on json tree shaking as unit test in `packages/vite/src/node/__tests__/build.spec.ts`?